dcos/packagemanager.py

```diff
@@ -41,6 +41,8 @@ def check_for_cosmos_error(*args, **kwargs):
             logger.debug("Error: {}".format(response.json()))
             error_msg = _format_error_message(response.json())
             raise DCOSException(error_msg)
+        elif (response.status_code / 100) != 2:
```

Comment:
I don't think this does what you meant to do, since Python 3 a division between 2 ints returns a float (cf. https://www.python.org/dev/peps/pep-0238/).

Reply:
Thanks! I casted the result of the division to an int. I don't this is the most idiomatic way to do it.

Reply:
Probably `range` is the idiomatic way. E.g. `reponse.status_code not in range(200, 300)`

Reply:
I'd suggest `if 200 <= response.status_code < 300:` which I find easier to read.